### PR TITLE
Test/release process

### PR DESCRIPTION
**Pull request by aurivus-ph**

### CODE REVIEW

File: ci/build-wheel-deps.sh [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
-    https://download.savannah.gnu.org/releases/freetype/freetype-2.10.4.tar.gz
-    https://github.com/harfbuzz/harfbuzz/archive/2.6.4.tar.gz
-)
+curl -L "https://download.savannah.gnu.org/releases/freetype/freetype-2.12.1.tar.gz" -o freetype-2.12.1.tar.gz
+tar zxf freetype-2.12.1.tar.gz
+pushd freetype-2.12.1
+./configure
+make -j $(nproc)
+make install
+popd
```
Comment (jwiggins) [inline ci/build-wheel-deps.sh:18]: Yeah. Sometimes doing the simplest thing is the best. DRY was not helping before 😄

File: ci/build-wheel-deps.sh [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
-    make
-    make install
-    popd
-done
+curl -L "https://github.com/harfbuzz/harfbuzz/archive/refs/tags/2.6.8.tar.gz" -o harfbuzz-2.6.8.tar.gz
+tar zxf harfbuzz-2.6.8.tar.gz
+pushd harfbuzz-2.6.8
+
+# missing macro in the locally installed automake
+cp /usr/share/aclocal/pkg.m4 /usr/local/share/aclocal
```
Comment (jwiggins) [inline ci/build-wheel-deps.sh:25]: Should this be relayed upstream to `RalfG/python-wheels-manylinux-build`?

Comment (aurivus-ph) [reply ci/build-wheel-deps.sh:25]: This seems to be a known issue in the base manylinux image: https://github.com/pypa/manylinux/issues/731

Comment (jwiggins) [reply ci/build-wheel-deps.sh:25]: 2020... 😬

**Pull request closed without merging**